cukinia/hypervisor_security_tests.d/mitigations.conf

```diff
@@ -0,0 +1,11 @@
+# Copyright (C) 2021, RTE (http://www.rte-france.com)
+# SPDX-License-Identifier: Apache-2.0
+
+cukinia_log "$(_colorize yellow "--- Check CPU vulnerabilities ---")"
+
+for i in /sys/devices/system/cpu/vulnerabilities/*; do  
+
+	as "ANSSI-BP-028 - System vulnerabilities ${i}" not cukinia_cmd\
```

Comment:
Not sur of the tag ANSSI-BP-028. Any idea ?

Reply:
Use SEAPATH-XXX